mobify/progressive-web-scaffold
# Disallow global jquery and jest

Removes the ESLint `jquery` and `jest` environments globally and instead relies on explicit configuration in files where global variables of these environments are necessary.

The main benefit of this PR is to enforce the use of our internal `$` variable returned by the SDK's `jqueryResponse` function, rather than the global `$` bound to `window` in browsers.

## Changes
- Removed the ESLint env settings.
- Updated all test files accordingly.

## How to test-drive this PR
- Run `npm run test:js`.
  - Verify that it succeeds.
- Open any file and try to use `$`.
  - Verify that ESLint reports an error.
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
